udata_front/theme/gouvfr/assets/js/components/utils/DeleteButton/DeleteUserButton.vue

```diff
@@ -71,15 +86,14 @@ const modalId = getRandomId("delete-modal");
 const modalTitleId = modalId + "-title";
 
 async function deleteObject() {
-  return deleteUser(props.user)
+  return deleteUser(props.id)
   .then(() => window.location.reload())
   .catch(() => toast.error(t("An error occured during the deletion.")));
 };
 
 function deleteObjectWithoutMail() {
```

Comment:
functions could have a better name because object is a strange way to call users